### DISCUSSION TITLE
Trim en los inputs

### DISCUSSION THREAD

**Discussion by NicoCroce** [TiendaNube/nimbus-design-system#186] [Ideas]
Buenas! Estamos teniendo algunos inconvenientes en inputs que pegan contenido. Muchas veces se generan espacios antes o después del texto. Si bien esto depende de la fuente en donde se copió la info, lo podemos trimear? 

Dejo el [issue](https://nuvemshop.slack.com/archives/C01CGQ21B0W/p1691769653777659?thread_ts=1691757562.608199&cid=C01CGQ21B0W) abierto. 

Muchas gracias! 👍

**Comment by juanchigallego** [comment]
Hola @NicoCroce ! Vamos a llevar este ajuste para modificar el componente y publicamos una nueva versión. Cuando esté lista dejamos la versión nueva para que actualicen.
